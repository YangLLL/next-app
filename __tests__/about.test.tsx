import { render } from '@testing-library/react'
import About from  '@/pages/about'

describe('Home', () => {
    it('render Home correctly', () => {
       const { container } = render(<About />)
       expect(container).toMatchSnapshot()
    })
})