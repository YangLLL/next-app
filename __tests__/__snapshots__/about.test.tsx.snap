// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render Home correctly 1`] = `
<div>
  <div>
    <p>
      About page
    </p>
  </div>
</div>
`;
